nautobot/ui/src/views/Login.js

```diff
@@ -57,9 +52,3 @@ export default function Login() {
         </Box>
     );
 }
-
-// TODO: This should be all that's needed to support SSO backends but doesn't work well with NodeJS dev mode for the time being
```

Comment:
Is this fixed now?

Reply:
We need to redesign the login view, not in the scope of this PR.